Add an example of `neo4j-admin database upload` for Windows users (#1700)
neo4j/docs-operations: modules/ROOT/pages/tools/neo4j-admin/upload-to-aura.adoc

.................... 100%
Your data was successfully pushed to Aura and is now running.
----

> In Windows, do not forget to use double backslashes `//` in the path file.

Reviewer: ```suggestion
On Windows, the backslashes `\` in the file paths must be escaped with another backslash.
```

For example:

[source, shell,role=nocopy]
----

> bin/neo4j-admin database upload dbname --from-path=c://db-dump-file// --to-uri=<neo4j+s://your-aura-instance-id.databases.neo4j.io> --overwrite-destination=true

Reviewer: ```suggestion
bin\neo4j-admin database upload dbname --from-path=c:\\db-dump-file\\ --to-uri=<neo4j+s:\\your-aura-instance-id.databases.neo4j.io> --overwrite-destination=true
```
